styles.css

```diff
@@ -33,17 +31,59 @@ body {
   list-style: none;
 }
 
+#nav-hidden {
+  display: none;
+}
+
 .menu {
-  height: 24px;
-  width: 24px;
+  height: 2.4rem;
+  width: 2.4rem;
+}
+
+@media screen and (min-width: 768px) {
+  .nav-bar {
+    display: none;
+  }
+
+  #nav-hidden {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    gap: 1rem;
+  }
+
+  .nav__list a,
+  li {
+    list-style: none;
+    font-weight: 600;
+    font-size: 1.5rem;
+    line-height: 2rem;
+    color: #344563;
+  }
 }
 
 .hero {
   background-image: url(/images/background.svg);
   background-repeat: no-repeat;
-  background-size: contain;
-  height: 726px;
-  margin-bottom: -50px;
+  background-size: 100%;
+  background-position: center;
+  height: 72.6rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+@media screen and (min-width: 768px) {
+  .hero {
+    background-image: none;
+  }
+
+  body {
+    background-image: url(/images/background-desktop.svg);
```

Comment:
Your project looks great. I commend your attention to detail. Kindly modify your paths specified in the project to depict relative rather than absolute paths. This is to avoid misbehaviour in hosting or other scenarios you might not expect like below:

![image](https://user-images.githubusercontent.com/20554583/193272722-274b9662-3fea-4ee8-9ad7-d35f1bf34056.png)

E.g Instead of `/images/background-desktop.svg` you may use `./images/background-desktop.svg`